frontend/src/components/EmployeeModal/EmployeeModal.tsx

```diff
@@ -142,6 +142,18 @@ const EmployeeModal = ({ existingEmployee }: EmployeeModalProps) => {
     refresh: () => Promise<void>,
     table: string
   ) => {
+   fetch(
+      endpoint,
+      withDefaults({
+        method: 'PUT',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(employeeData),
+      })
```

Comment:
the fetch request is already being handled in `updatedEmployee`, dont need to do it twice. you can do something like 
```
then((res) => {
  refresh();
  setToast(true)
  return res.json()
})
```